starter/script.js

```diff
@@ -1 +1,158 @@
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+function listThirdAnswers(){
+  var innerElement = "";
+  for (i = 0; i < thirdQuestion.length; i++)
+  {
+    innerElement = innerElement + "<li><a id=ans_" + (i + 7) + " href='#' onClick=listOfAnswers(this)>" + thirdQuestion[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+function listOfQuestions (elm){
+  switch(elm.id){
+    case "quest_1":
+      listFirstAnswers();
+      break;
+    case "quest_2":
+      listSecondAnswers();
+      break;
+    case "quest_3":
+      listThirdAnswers();
+      break;
+    case "quest_4":
+      listFourthAnswers();
+      break;
+    case "quest_5":
+      listFifthAnswers();
+      break;
+    case "quest_6":
+      listSixthAnswers();
+      break;
+    default:
+      break;
+  }
+};
+
+function listOfPathOne() {
+  var innerElement = '';
+  for(i=0; i < pathOne.length; i++) {
+    innerElement = innerElement + "<li><a id=quest_" + (i + 1) + " href='#' onClick=listOfQuestions(this)>" + pathOne[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+function listOfPathTwo() {
+  var innerElement = '';
+  for(i=0; i < pathTwo.length; i++) {
+    innerElement = innerElement + "<li><a id=quest_" + (i + 1) + " href='#' onClick=listOfQuestions(this)>" + pathTwo[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+//this is for the second question:**********************************************************************
+
+function listSecondAnswers(){
+  var innerElement = "";
+  for (i = 0; i < secondQuestion.length; i++)
+  {
+    innerElement = innerElement + "<li><a id=ans_" + (i + 4) + " href='#' onClick=listOfAnswers(this)>" + secondQuestion[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+function listOfQuestions(elm){
+  switch(elm.id){
+    case "quest_1":
+      listFirstAnswers();
+      break;
+    case "quest_2":
+      listSecondAnswers();
+      break;
+    case "quest_3":
+      listThirdAnswers();
+      break;
+    default:
+      break;
+  }
+};
+
+function listOfPath() {
+  var innerElement = '';
+  for(i=0; i < pathOne.length; i++) {
+    innerElement = innerElement + "<li><a id=quest_" + (i + 1) + " href='#' onClick=listOfQuestions(this)>" + pathOne[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+//this is for the third question:**********************************************************************
+
+function listThirdAnswers(){
+  var innerElement = "";
+  for (i = 0; i < secondQuestion.length; i++)
+  {
+    innerElement = innerElement + "<li><a id=ans_" + (i + 7) + " href='#' onClick=listOfAnswers(this)>" + thirdQuestion[i] + "</a></li>";
+  }
+  document.getElementById('main#1').innerHTML = innerElement;
+};
+
+function listOfQuestions(elm){
```

Comment:
you've defined this function elsewhere in your code, be careful to overwrite your own functions!